scheduler-driver/src/main/java/gov/nasa/jpl/aerie/scheduler/simulation/ResumableSimulationDriver.java

```diff
@@ -109,10 +125,14 @@ public int getCountSimulationRestarts(){
 
   @Override
   public void close() {
+    logger.info("Closing sim");
```

Comment:
Minor, can ignore: since there's already an info-level log for restarting the driver, this line only needs to be at the debug or trace level
```suggestion
    logger.debug("Closing Simulation Driver");
```